src/Manager/Indices.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+namespace WPML\ElasticPress\Manager;
+
+use ElasticPress\Elasticsearch;
+use ElasticPress\Indexables;
+use ElasticPress\Indexable;
+
+use WPML\ElasticPress\Traits\TranslateLanguages;
+
+class Indices {
+
+	use TranslateLanguages;
+
+	/** @var Elasticsearch */
+	private $elasticsearch;
+
+	/** @var Indexables */
+	private $indexables;
+
+	/** @var array */
+	private $activeLanguages;
+
+	/** @var string */
+	private $defaultLanguage;
+
+	/** @var string */
+	private $currentIndexLanguage = '';
+
+	/**
+	 * @param Elasticsearch $elasticsearch
+	 * @param Indexables    $indexables
+	 * @param array         $activeLanguages
+	 * @param string        $defaultLanguage
+	 */
+	public function __construct(
+		Elasticsearch $elasticsearch,
+		Indexables    $indexables,
+		$activeLanguages,
+		$defaultLanguage
+	) {
+		$this->elasticsearch   = $elasticsearch;
+		$this->indexables      = $indexables;
+		$this->activeLanguages = $activeLanguages;
+		$this->defaultLanguage = $defaultLanguage;
+	}
+
+	public function addHooks() {
+		add_filter( 'ep_index_name', [ $this, 'filterIndexName' ], 10, 1 );
```

Comment:
The last argument `1` is already the default, you can get rid of it.

Reply:
Removed all default args that were explicitly set on `add_action` and `add_filter` calls.